src/main/java/org/glassfish/spec/Spec.java

```diff
@@ -59,6 +49,12 @@
      */
     private JarFile jar;
 
+    /**
+     * The Spec mode (<code>"javaee"</code> or <code>"jakarta"</code>).
+     * Default value is {@link SpecMode.JAVAEE}.
```

Comment:
Since this is the Eclipse version of the plugin, shouldn't the default be "jakarta"?
Maybe also change the version number of the plugin to 2.0 to reflect this change in defaults?

Reply:
Yes, you are right. Fixed.